inst/include/smc-exception.h

```diff
@@ -38,7 +38,14 @@
 #define SMCX_MISSING_HISTORY 0x0010
 ///Exception thrown if an attempt is made to instantiate a class of which a single instance is permitted more than once.
 #define SMCX_MULTIPLE_INSTANTIATION 0x1000
-
+///Exception thrown if an attempt is made to instantiate a class of which a single instance is permitted more than once.
+#define SMCX_MULTIPLE_INSTANTIATION 0x1000
+//Exception thrown if an attempt is made to instantiate a class of which a single instance is permitted more than once.
+#define SMCX_MULTIPLE_INSTANTIATION 0x1000
```

Comment:
There seems to be some duplication here? (41-42; 43-44 look rather similar up to a /).

Reply:
Done!